### PR TITLE
Parallelize Transit Initializer

### PR DESCRIPTION
**Pull request by SHaaD94**
https://github.com/LBNL-UCB-STI/beam/issues/2544

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2576)

### CODE REVIEW

**Comment by SHaaD94** [comment]
Test!

**Comment by SHaaD94** [comment]
Test!

**Comment by SHaaD94** [comment]
Test!

File: src/main/scala/beam/router/TransitInitializer.scala [hunk 1]
```diff
@@ -44,10 +45,10 @@ class TransitInitializer(
    * be used to decide what type of vehicle to assign
    *
    */
-  def initMap: Map[Id[BeamVehicle], (RouteInfo, ArrayBuffer[BeamLeg])] = {
+  def initMap: ParMap[Id[BeamVehicle], (RouteInfo, Seq[BeamLeg])] = {
```
Comment (REASY) [inline src/main/scala/beam/router/TransitInitializer.scala:48]: Let's bring back the original `Map` type. When data is ready, you can use `.seq` to convert it to normal Map

Comment (SHaaD94) [reply src/main/scala/beam/router/TransitInitializer.scala]: fixed

**Comment by SHaaD94** [comment]
Test!

**Pull request merged**